src/Abc.Zebus/Hosting/HostInitializer.cs

```diff
@@ -14,6 +14,8 @@ public abstract class HostInitializer
         /// </summary>
         public virtual int Priority => 0;
 
+        public virtual bool CallAfterStartBeforeMessageDeliveryHasStarted => false;
```

Comment:
How will you hook this into our host? I don't see an easy way.

Reply:
We could call `AfterStart` on the `StartedButNotDeliveringMessages` event instead of directly after calling `Start`